wfe/web-front-end.go

```diff
@@ -170,14 +171,14 @@ func (wfe *WebFrontEndImpl) HandleFunc(mux *http.ServeMux, pattern string, h fun
 		}
 		response.Header().Set("Access-Control-Allow-Origin", "*")
 
-		switch request.Method {
-		case "HEAD":
+		// Return a bodyless response to HEAD for any resource that allows GET.
+		if _, ok := methodsOK["GET"]; ok && request.Method == "HEAD" {
 			// We'll be sending an error anyway, but we
 			// should still comply with HTTP spec by not
 			// sending a body.
 			response = BodylessResponseWriter{response}
-		case "OPTIONS":
-			// TODO, #469
+			h(response, request)
```

Comment:
You're right, looks like this introduced some bugs that weren't caught by testing, I'm sorry. Will fix today.